dprint.h

```diff
@@ -340,49 +353,27 @@ static inline char *dp_log_level_str(int log_level)
 		#define LM_GEN2( _facility, _lev, fmt, ...) \
```

Comment:
`LM_GEN1` and `LM_GEN2` is used by `xlog(...)` which I address by this PR. They are also called from a few other places with debug level (like memory stat dump) and these are also will use the 3.2 behavior (no prefix) with this changes which is excepted.